charts/kube-state/templates/kube-state-deployment.yaml

```diff
@@ -1,6 +1,10 @@
 ################################
 ## Kube State Deployment
 #################################
+{{- $useRoles := or .Values.global.features.namespacePools.enabled .Values.global.singleNamespace }}
+{{- $namespaces := .Values.global.features.namespacePools.namespaces.names }}
+{{- $namespaces = append $namespaces .Release.Namespace }}
```

Comment:
@danielhoherd only for single namespace we need .Release.Namespace to be added here 
i can create a condition like
```
{{ if .Values.global.singleNamespace  }}
{{- $namespaces = append $namespaces .Release.Namespace }}
{{ else }}
{{- $namespaces = append $namespaces }}
{{ end }}
```